pkg/cmd/build/new.go

```diff
@@ -14,25 +14,24 @@ import (
 )
 
 func NewCmdBuildNew(f *factory.Factory) *cobra.Command {
-	var message string
-	var commit string
 	var branch string
+	var commit string
+	var message string
+	var pipeline string
 	var web bool
 
 	cmd := cobra.Command{
 		DisableFlagsInUseLine: true,
-		Use:                   "new [pipeline] [flags]",
-		Short:                 "Creates a new pipeline build",
-		Args:                  cobra.MaximumNArgs(1),
+		Use:                   "new [flags]",
+		Short:                 "Create a new build",
+		Args:                  cobra.NoArgs,
 		Long: heredoc.Doc(`
-			Creates a new build for the specified pipeline and output the URL to the build.
-
-			The pipeline can be a {pipeline_slug} or in the format {org_slug}/{pipeline_slug}.
-			If the pipeline argument is omitted, it will be resolved using the current directory.
+			Create a new build on a pipeline.
+			The web URL to the build will be printed to stdout.
 		`),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			resolvers := resolver.NewAggregateResolver(
-				resolver.ResolveFromPositionalArgument(args, 0, f.Config),
+				resolver.ResolveFromFlag(pipeline, f.Config),
```

Comment:
There is no build argument to this command. So I thought making the pipeline name optional instead of an arg would make sense. Any thoughts?

Reply:
I am fine with that behaviour.